RBS.Auth.WebApi/RBS.Auth.WebApi/Startup.cs

```diff
@@ -1,84 +1,85 @@
+    public IConfiguration Configuration { get; }
+    public IWebHostEnvironment Environment { get; }
+
+    // This method gets called by the runtime. Use this method to add services to the container.
+    public void ConfigureServices(IServiceCollection services)
+    {
+        services.ConfigureDatabase(Configuration);
+
+        services.AddAutoMapper(typeof(Startup));
+
+        ConfigureInternalServices(services);
 
-            services.AddControllers();
+        services.AddControllers();
 
-            services.Configure<AuthOptions>(Configuration.GetSection("Auth"));
+        services.Configure<AuthOptions>(Configuration.GetSection("Auth"));
 
-            if (!Environment.IsProduction())
+        if (!Environment.IsProduction())
+            services.AddSwaggerGen(options =>
             {
-                services.AddSwaggerGen(options =>
-                {
-                    options.SwaggerDoc("v1", new OpenApiInfo { Title = "RBS.Auth.WebApi", Version = "v1" });
-                });
-            }
-        }
+                options.SwaggerDoc("v1", new OpenApiInfo {Title = "RBS.Auth.WebApi", Version = "v1"});
+            });
+    }
 
-        // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
-        public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
+    // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
+    public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
+    {
+        if (env.IsDevelopment())
         {
-            if (env.IsDevelopment())
+            app.UseSwagger();
+            app.UseSwaggerUI(opt =>
             {
-                app.UseSwagger();
-                app.UseSwaggerUI(opt =>
-                {
-                    opt.SwaggerEndpoint("/swagger/v1/swagger.json", "v1");
-                    opt.RoutePrefix = string.Empty;
-                });
-                app.UseDeveloperExceptionPage();
-            }
+                opt.SwaggerEndpoint("/swagger/v1/swagger.json", "v1");
+                opt.RoutePrefix = string.Empty;
+            });
+            app.UseDeveloperExceptionPage();
+        }
 
 
-            app.UseHttpsRedirection();
+        app.UseHttpsRedirection();
 
-            app.UseRouting();
+        app.UseRouting();
 
-            app.UseAuthorization();
+        app.UseAuthorization();
 
-            app.UseEndpoints(endpoints =>
-            {
-                endpoints.MapControllers();
-            });
-        }
-    
-        private void ConfigureInternalServices(IServiceCollection services)
-        {
-            services.AddScoped<IJWTTokenService, JWTTokenService>();
-            services.AddScoped<IAuthenticateService, AuthenticateService>();
-        }
+        app.UseEndpoints(endpoints => { endpoints.MapControllers(); });
+    }
+
+    private void ConfigureInternalServices(IServiceCollection services)
```

Comment:
Consider to move this method to extensions with meaningful naming, for instance `services.AddAuthExtensions()`